packages/core/__mocks__/MockAdapter.ts

```diff
@@ -0,0 +1,61 @@
+import * as Sentry from '../index';
+
+export interface IMockAdapterOptions {
+  testOption?: boolean;
+}
+
+export class MockAdapter implements Sentry.IAdapter {
+  constructor(
+    client: Sentry.Client,
+    public options: IMockAdapterOptions = { testOption: false }
+  ) {
+    return this;
+  }
+
+  public install(): Promise<boolean> {
+    return Promise.resolve(true);
+  }
+
+  public setOptions(options: IMockAdapterOptions) {
+    // We need nothing here
+    return this;
+  }
+
+  public captureException(exception: Error): Promise<Sentry.Event> {
+    return Promise.resolve(new Sentry.Event());
+  }
+
+  public captureMessage(message: string): Promise<Sentry.Event> {
```

Comment:
Could use `async` here and in the other methods in this class. But I also like the verbosity, if that's what you're after.

Reply:
Tried to rewrite it, but because I use a local import here instead of package import it fails with:
```
[ts] Return type of public method from exported class has or is using name 'Event' from external module "/Users/haza/Projects/HazAT/raven-js/packages/core/lib/Interfaces/Event" but cannot be named.
```

This is the way it should be imported `import { Client, Event, IAdapter, IBreadcrumb, IUser } from '@sentry/core';` then it works.